libgit/rpc.go

```diff
@@ -170,8 +172,10 @@ func (rh *RPCHandler) scheduleCleaning(folder keybase1.Folder) {
 	time.AfterFunc(minDeletedAgeForCleaning+1*time.Second, func() {
 		log := rh.config.MakeLogger("")
 
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
 		ctx, gitConfig, tlfHandle, tempDir, err := rh.getHandleAndConfig(
-			context.Background(), folder)
+			ctx, folder)
```

Comment:
It's a better pattern to use the `context.WithCancel`, so we should switch anyway, but I don't see the leak here.

Reply:
Eventually `getHandleAndConfig` calls `libgit.Init()`, which calls `libkbfs.NewContextWithCancellationDelayer()`.

Reply:
Fair enough. I guess the contract of any ctx is that it will eventually be canceled, so we should just always be doing this.